src/app/cli/src/coda.ml

```diff
@@ -589,8 +589,12 @@ let daemon logger =
            (Async.Scheduler.long_cycles
               ~at_least:(sec 0.5 |> Time_ns.Span.of_span_float_round_nearest))
            ~f:(fun span ->
+             let secs = Time_ns.Span.to_sec span in
              Logger.debug logger ~module_:__MODULE__ ~location:__LOC__
-               "Long async cycle, %0.01f seconds" (Time_ns.Span.to_sec span) ) ;
+               "Long async cycle, %0.01f seconds" secs ;
```

Comment:
One of @yourbuddyconner's asks was to put the secs into the metadata so that we can pull it out in kibana. Can you do that as well? Maybe name it something like "time_elapsed"?